install.sh

```diff
@@ -192,8 +193,8 @@ if [ ! -f $BUILD/progress/ghc ]; then
 fi
 
 if [ ! -f $BUILD/progress/cabal-install ]; then
-  run .                  ghcup install-cabal
-  run .                  cabal update
+  run .                  ghcup install-cabal 2.4.1.0
+  run .                  cabal update --index-state='2019-09-28T01:46:00Z'
```

Comment:
You probably also want to remove the `cabal update` in `build.sh`

Reply:
Done